### DISCUSSION TITLE
Setting padding on menus?

### DISCUSSION THREAD

**Discussion by chamons** [emilk/egui#287] [Q&A] [answered]
I would like this menu to breath a bit more with space in the X-axis (left/right of options).

![image](https://user-images.githubusercontent.com/6392201/114129100-84dc3200-98c3-11eb-8df4-62d1ddf84e8d.png)

I tried `ui.style_mut().spacing.item_spacing.x += 40.0` and other options, and was unable to get any spacing.

`ui.add_space(2.0);` was enough to get vertical spacing.

**Comment by emilk** [comment, marked as answer]
This is currently hard-coded at https://github.com/emilk/egui/blob/master/egui/src/containers/frame.rs#L64 so no way to change it. Should be easy enough to add an option for it in `Spacing` if you would like to contribute a PR :)

**Reply by chamons** [reply]
I might do that, thanks for the pointer.

**Reply by chamons** [reply]
It's been more than a year, but I finally got around to doing that PR: https://github.com/emilk/egui/pull/2036
